test/objects.coffee

```diff
@@ -44,6 +44,15 @@ pluck = (x) -> x.a
 eq 100, pluck pluck pluck a: a: a: 100
 
 
+test "Object literal spread", ->
+  obj1 =
+    a: 1
+    b: 2
+  # obj2 = "#{{obj1..., b: 3}}"
+  t1 = CoffeeScript.compile "{obj1..., b: 3}", bare: yes
+  t1 = t1.replace(/\s+/g, "")
+  eq t1, "({...obj1,b:3});"
```

Comment:
It is our policy to only support ES features that have reached stage 4, and are supported in the Node runtime (modules are the big exception to this). Therefore the test would compile to the finalized ES syntax, and Node should be able to run it.